scss/bitstyles/ui/forms.stories.mdx

```diff
@@ -0,0 +1,234 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="UI/Data/Forms" />
+
+# Forms
+
+**See inputs & labels for examples of using inputs of various types.**
+
+When using form elements in a form, they should always be wrapped in a list element, as this indicates to the user that all the elements are closely related to each other. This also gives us the opportunity to apply some layout, such as adding space between elements.
+
+Form elements will take 100% of the available width, so you’ll likely want to constrain them within a grid or flex layout system to make that more readable and usable.
+
+Here are some examples using the grid utility classes to create common form layouts.
+
+## Login form
+
+<Canvas>
+  <Story name="Login form">
+    {`
+      <div class="a-content a-content--small">
+        <form class="a-card@m u-padding-xl@m" method="POST" action="">
+          <h3>Login</h3>
+          <ul class="a-list-reset u-margin-xl--bottom">
+            <li class="u-margin-m--bottom">
+              <label for="email">Email</label>
+              <input id="email" type="email" autocomplete="email" />
+            </li>
+            <li class="u-margin-m--bottom u-flex u-flex--col">
+              <label for="password">Password</label>
+              <input id="password" type="password" autocomplete="current-password" />
```

Comment:
**TIL* `current-password` 💡

Reply:
I love these, and they seemingly just appeared sometime, with little fanfare. The [MDN page](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) is of course great